sdk/core/core-amqp/samples/send.ts

```diff
@@ -1,26 +1,64 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for license information.
+/*
+  Copyright (c) Microsoft Corporation. All rights reserved.
+  Licensed under the MIT Licence.
+
+  This sample demonstrates how to send messages/events to Service Bus/Event Hubs.
+*/
+
+import {
+  ConnectionContextBase,
+  CreateConnectionContextBaseParameters,
+  CbsResponse,
+  ConnectionConfig,
+  TokenType,
+  SharedKeyCredential
+} from "@azure/core-amqp";
 
-import * as dotenv from "dotenv";
-dotenv.config(); // Optional for loading environment configuration from a .env (config) file
 import { Sender, SenderOptions, EventContext, Message, Delivery } from "rhea-promise";
-import { authenticate, connectionContext, connectionConfig, path } from "./cbsAuth";
+
+// Define connection string and related entity path here
+const connectionString = "";
+const path = "";
+
+const connectionConfig = ConnectionConfig.create(connectionString, path);
+const parameters: CreateConnectionContextBaseParameters = {
```

Comment:
Let's pass this directly to `ConnectionContextBase.create()` function. Having it separate here adds a step for the user to understand what exactly the `CreateConnectionContextBaseParameters` is.
Also, lets just pass the config. That is the only mandatory argument right?

Same for all other samples

Reply:
`connectionProperties` is a mandatory argument in `CreateConnectionContextBaseParameters` and also `product`, `userAgent` and `version` are the mandatory arguments  in `ConnectionProperties`

And I think it looks clearer this way but if we don't want to show `CreateConnectionContextBaseParameters` to user than we can simply define `parameters` like this 
```
const parameters = {
  config: connectionConfig,
  connectionProperties: {
    product: "MSJSClient",
    userAgent: "/js-core-amqp",
    version: "0.1.0"
  }
};
```